test/unit/org/apache/cassandra/index/sai/utils/KeyRangeConcatIteratorTest.java

```diff
@@ -192,7 +192,6 @@ public void testEmptyThenManyNonEmpty()
         IntStream.range(10, 20).forEach(value -> builder.add(build(value)));
 
         KeyRangeIterator keyIterator = builder.build();
```

Comment:
nit: Static analysis tools will complain about us not closing the iterator here, but we don't care...it's using `LongIterator`. Could throw `@SuppressWarnings("resource")` on it if we want to get rid of the noise.